### PR TITLE
[12.0][MIG] contract_commission migration

### PR DESCRIPTION
**Pull request by marcelsavegnago**
cc @eduaparicio

### CODE REVIEW

**Review by eduaparicio — APPROVED** [review]
Functional review. Everything looks good!

**Review by jpcarassato — APPROVED** [review]
Test Ok!

**Comment by marcelsavegnago** [comment]
@pedrobaeza Can you take a look at this travis error and help me please?
https://travis-ci.com/github/OCA/commission/jobs/424917683

**Comment by pedrobaeza** [comment]
It seems Travis got stuck. Have you tried to restart the build? I have launched it to see.

File: contract_commission/tests/test_contract_commission.py [hunk 1]
```diff
@@ -50,10 +74,16 @@ def test_invoice_commissions(self):
...
         invoice_lines = self.env['account.invoice.line'].search(
-            [('account_analytic_id', '=', self.contract.id)]
+            [('contract_line_id', 'in',
+              contracts.mapped('contract_line_ids').ids)]
         )
         self.assertEqual(len(invoice_lines), 1)
         self.assertFalse(invoice_lines.agents)
+
+    def test_prepare_invoice_line(self):
+        result = self.invoice._prepare_invoice_line()
```
Comment (rousseldenis) [inline contract_commission/tests/test_contract_commission.py]: @marcelsavegnago Please see travis error. Missing arguments here

**Comment by marcelsavegnago** [comment]
ping @rousseldenis

**Comment by rousseldenis** [comment]
@marcelsavegnago Could you squash your commits ?

**Comment by marcelsavegnago** [comment]
> @marcelsavegnago Could you squash your commits ?

@rousseldenis done.

**Comment by marcelsavegnago** [comment]
ping @rousseldenis

**Review by rvalyi — APPROVED** [review]
LGTM

**Comment by marcelsavegnago** [comment]
up

**Pull request merged**